packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -209,9 +209,9 @@ const PoolCard = decorate([
           {state.poolHosts !== undefined && !state.isPoolXostorCompatible && (
             <div className='text-danger'>
               {/* FIXME: add link of the documentation when ready */}
-              <a href='#' rel='noreferrer' target='_blank'>
+              {/* <a href='#' rel='noreferrer' target='_blank'>
```

Comment:
Why not simply remove it?

Reply:
It will be reintroduced once the documentation is available.